vars/PipelineNode.groovy

```diff
@@ -99,6 +99,7 @@ def call(body) {
           createNodeComposeNpmAuditEnv(config, './npm-audit.json')
           sh(script: "docker-compose -f npm-audit.json up --no-start")
           sh(script: "docker-compose -f npm-audit.json run main npm audit --production --audit-level=high > ci-outputs/npm-audit/audit.json")
+          sh(script: "cat ci-outputs/npm-audit/audit.json")
```

Comment:
Spíš bych to udělala tak, že bych to přes `tee` sypala jak do souboru, tak na stdout :-)

Reply:
Změň, jestli chceš :)